core.js

```diff
@@ -1,25 +1,28 @@
-'use strict';
-const Token = require('token-types');
-const strtok3 = require('strtok3/lib/core');
-const {
+import Token from 'token-types';
+import {
+	fromStream as tokenizerFromStream,
+	fromBuffer as tokenizerFromBuffer,
+	EndOfStreamError,
+} from 'strtok3/lib/core.js';
```

Comment:
@Borewit This kind of deep import is discouraged with ESM. Instead, you should use the `exports` field in package.json to properly export it. Can you do that? It will require a major release as implicit sub-exports like this will no longer be available then.

https://nodejs.org/api/packages.html#packages_subpath_exports

Reply:
I will try to find a solution with ESM export.